spec/fixtures/storage_root03/moab_storage_trunk/bb/077/xv/1376/bb077xv1376/v0001/data/metadata/technicalMetadata.xml

```diff
@@ -0,0 +1,153 @@
+<technicalMetadata xmlns:jhove="http://hul.harvard.edu/ois/xml/ns/jhove" xmlns:mix="http://www.loc.gov/mix/v10" xmlns:textmd="info:lc/xmlns/textMD-v3" objectId="druid:bb077xv1376" datetime="2019-07-04T10:25:13Z">
```

Comment:
we don't preserve technical metadata anymore, do we still need this file?

Reply:
Removed, and tests still pass